### PR TITLE
Create Rule “mockups/rule”

### PR DESCRIPTION
**Pull request by hveraus**
Automatically generated by Netlify CMS

### CODE REVIEW

File: rules/mockups/rule.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+:::  info
+**Incorporate branding:** It's important to incorporate the brand's visual identity into the design of a product! This should include the brand's given color scheme, typography, and logo at a minimum.
+:::
+
+::: good
+![Figure: Good example – High-fidelity mockup for ssw internship page](high-fidelity-mockup.jpg)
+:::
+
+### Interactive Mockups
```
Comment (jaydenalchin) [inline rules/mockups/rule.md:69]:
```suggestion
### Interactive Mockups (a.k.a. Prototypes)
```

**Review by jaydenalchin — COMMENTED** [review]
Looks great @hveraus ! 

1. Please add this to the Better Designers category 
2. Consider renaming this rule to something like "Do you know the different types of mockups?" or "Do you know the difference between low and high fidelity mockups?" 
3. I think the storyboarding rule should have more of the focus on capturing specs. 
4. Adding a suggestion for a small addition making it more clear that Interactive Mockup = Prototype

Thanks!

File: rules/mockups/rule.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+guid: 2b8c7067-4d59-49b2-a448-4b746b11383a
+---
+Many user requirements can be best encapsulated in visual mockups. A design mockup is  usually developed at the specification review phase to illustrates the layout, structure, and content of a product or interface before it is developed. A mockup can become a prototype over time but they tend to come earlier in the design process and are static images rather than interactive testing tools.
+
+<!--endintro-->
+
+There are 3 levels of mockups:
+
+1. **Low-fidelity Mockups** 
+2. **High-fidelity Mockups** (recommended)
```
Comment (tiagov8) [inline rules/mockups/rule.md:24]: I don't think we can recommend one or another... it's based on the project requirements

File: rules/mockups/rule.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  - title: Jayden Alchin
+    url: https://www.ssw.com.au/people/jayden-alchin
+  - title: Geordie Robinson
+    url: https://ssw.com.au/people/geordie-robinson
+related:
+  - storyboards
+created: 2023-09-04T07:37:31.732Z
+guid: 2b8c7067-4d59-49b2-a448-4b746b11383a
+---
+Many user requirements can be best encapsulated in visual mockups. A design mockup is  usually developed at the specification review phase to illustrates the layout, structure, and content of a product or interface before it is developed. A mockup can become a prototype over time but they tend to come earlier in the design process and are static images rather than interactive testing tools.
```
Comment (tiagov8) [inline rules/mockups/rule.md:17]: We should say "mockups or prototypes" everywhere because these are different terms

File: rules/mockups/rule.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
+---
+type: rule
+title: Do you determine specifications with mockups?
+uri: mockups
```
Comment (tiagov8) [inline rules/mockups/rule.md:4]: URL should be:

mockups-and-prototypes

**Comment by tiagov8** [comment]
> Consider renaming this rule to something like "Do you know the different types of mockups?" or "Do you know the difference between low and high fidelity mockups?"

I think I prefer: "Do you know when to create mockups/prototypes?"

File: rules/mockups/rule.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
+---
+type: rule
+title: Do you know when to create mockups/prototypes?
```
Comment (jaydenalchin) [inline rules/mockups/rule.md:3]:
```suggestion
title: Do you know when to create mockups and prototypes?
```

File: rules/mockups/rule.md (hunk 4 above)
Comment (tiagov8) [reply rules/mockups/rule.md]: Done - updated folder name and URI

**Review by tiagov8 — APPROVED** [review]
Hi @hveraus

As per our conversation on Designers meeting, I am approving this PR since although not 100% by all designers, it's definitely better than what we had before.

**Pull request merged**